### DISCUSSION TITLE
Webdav authentication to infinicloud

### DISCUSSION THREAD

**Discussion by sageres1** [2fast-team/2fast#150] [Q&A]
Has anyone able to successfully authenticate to infinicloud? For some odd reason I am getting an error message, "The address to the WebDav service could not be found", yet if I were to try pasting from my browsers -- I get a login prompt. The service address is https://domi.teracloud.jp/dav/

**Comment by jp-weber** [comment]
The problem is that the status.php page of the webDAV service requires authorisation, which is not defined in the app, see new issue #151.

**Comment by dembala** [comment]
Hi. I get the same error with my own webdav (Synology NAS), i can't use 2fast for webdav access currently. I'm using it like this: https://servername.domain:port  I tried also putting the directories behind, no use, nothing is accepted, without https either. For me only local database works. Obviously the webdav service works for other clients (25+) in my infrastructure so nothing is wrong with webdav on NAS side.
